test.py

```diff
@@ -77,6 +77,17 @@ def __init__(self):
         from .relative_function_inside_class import a_third_function
         import pprint
 """},
+
+{'targets':
+     {'questionable': ['groucho', 'harpo', 'chico']},
```

Comment:
I would update the ordering of the questionable elements or change the tests to only check that the sets are equivalent. I'm not certain that we care about the ordering of the deps in depfinder at the moment, so laundering them through set might be ok.

Reply:
I changed the order and now it works fine